daemon/src/setup_contract.rs

```diff
@@ -54,6 +54,10 @@ use std::ops::RangeInclusive;
 use std::time::Duration;
 use xtra::prelude::MessageChannel;
 
+/// How long protocol waits for the next message before giving up
+// TODO: Move back to 60s after issue of actor test failures has been resolved.
```

Comment:
```suggestion
// TODO: Analyse why event sourcing refactor made us change the timeout from 60 to 70 secs. Why does it take so much longer now?
```

Let's see if the 70 secs plaster is going to make a difference. If so, we might keep them for the time being given the current code. 
Not sure we will then change them back to 60. If the 70 secs don't help we will have to analyse why it keeps failing.

Reply:
yeah, I still think we should analyse that, but there are more pressing issues (such as 0.4 release :) ). Hopefully this will do in the meantime - I used the first sentence of your suggestion, thanks!